### PR TITLE
Add accessibility attributes to close button

### PR DESCRIPTION
**Pull request by nosami**
on Debug Preview Visualizer Window. Fixes #648992

### CODE REVIEW

**Comment by nosami** [comment]
@monojenkins backport release-7.7

**Comment by nosami** [comment]
@monojenkins backport release-7.7

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger.PreviewVisualizers/PreviewVisualizerWindow.cs [hunk 1]
```diff
@@ -58,6 +59,7 @@ public PreviewVisualizerWindow (ObjectValue val, Gtk.Widget invokingWidget) : ba
 				InactiveImage = ImageService.GetIcon ("md-popup-close", IconSize.Menu),
 				Image = ImageService.GetIcon ("md-popup-close-hover", IconSize.Menu)
 			};
+			closeButton.SetCommonAccessibilityAttributes ("Preview visualizer close button", "Close button", null);
```
Comment (iainx) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger.PreviewVisualizers/PreviewVisualizerWindow.cs]: Close button should be translated, and should just read "Close", VoiceOver will already say it's a button

**Comment by iainx** [comment]
Oh, it has been translated, I didn't see that commit. It should still read "Close"

**Comment by slluis** [comment]
@nosami can you change the label to just "Close"?

**Comment by nosami** [comment]
@monojenkins backport release-7.7

**Pull request merged**